### PR TITLE
initial commit of authentication

### PR DESCRIPTION
**Pull request by Arpit-dotcom**
Made login, logout and signup page
image
image
image

### CODE REVIEW

File: authentication/authentication.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+.container{
+    margin: auto;
+    padding: 2rem;
+    background-color: white;
+    width: 20rem;
+}
+
+p{
+    margin: 0;
+}
```
Comment (bharati-21) [inline authentication/authentication.css:14]: These resets could be added to a stylesheet common to all pages. 👍🏽

File: authentication/login.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (bharati-21) [inline authentication/login.html:6]: You can add the meta author and description tags here.

**Review by bharati-21 — APPROVED** [review]
Looks good to be merged. 👍🏽

File: authentication/authentication.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+.img{
+    width: 3rem;
+    height: 3rem;
+}
+
+.signin{
+    border: 1px solid thistle;
```
Comment (rohanmathur91) [inline authentication/authentication.css:22]: have a variable for this color and use that

File: authentication/authentication.css [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    padding: .4rem 5.5rem;
+    text-align: center;
+    border-radius: 1rem;
+    text-decoration: none;
+    color: black;
```
Comment (rohanmathur91) [inline authentication/authentication.css:27]: same as above

File: authentication/authentication.css [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+.inp{
+    margin: 1rem 0;
+    width: 85%;
+    padding: .5rem 1rem;
+    border-radius: 1rem;
+    border: 1px solid black;
```
Comment (rohanmathur91) [inline authentication/authentication.css:35]: same as above

File: authentication/authentication.css [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    color: black;
+}
+
+.submit{
+    padding: .5rem;
+    border-radius: 1rem;
+    border: 1px solid black;
+    width: 95%;
+    background-color: brown;
+    color: white;
```
Comment (rohanmathur91) [inline authentication/authentication.css:46]: same as above

**Review by rohanmathur91 — APPROVED** [review]
Have a variable for every color you are using, rest looks fine.

File: authentication/authentication.css (hunk 1 above)
Comment (Arpit-dotcom) [reply authentication/authentication.css:14]: removed

**Pull request merged**